tests/core/mempool/test_mempool_manager.py

```diff
@@ -534,3 +534,36 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [sb1]
+
```

Comment:
I think we need a new test (maybe in benchmarks) that simulates the mempool running for a long time (several weeks)

Reply:
What do you think?

Reply:
it's a bit nebulous. We could have a test similar to the benchmark that causes churn through the mempool. but the test would still (presumably) be human inspection of how the process is doing.

Reply:
I made a program that churns transactions through the mempool for about a week worth of blocks ([here](https://github.com/Chia-Network/chia-blockchain/pull/14728)) and running that on this patch seems to reach the stable state (when the mempool is full). without leaking memory.

<img width="424" alt="image" src="https://user-images.githubusercontent.com/661450/222755952-9516d18a-4d8a-4bc6-8927-eccccd39dcff.png">